### ISSUE TITLE
Audio playback from Uint8List \ List<int> stream 

### ISSUE THREAD

**Issue by Zarainia** [alexmercerind/dart_vlc#84]
I saw this issue on the previous flutter_audio_desktop library (https://github.com/alexmercerind/flutter_audio_desktop/issues/32). Is this available in this library/will it be implemented?

**Comment by Gitzzq** [comment]
Yes, I think this will be a great feature to have. Sometimes the video/audio data just can not be put in a disk file, e.g. encrypted file can only be stored in memory for security reason.

**Comment by venus0707** [comment]
Yes, I think this will be a great feature to have
